src/components/scenes/WalletListScene.js

```diff
@@ -160,7 +191,9 @@ export const WalletListScene = connect(
     return {
       activeWalletIds,
       exchangeRates: state.exchangeRates,
-      wallets: state.ui.wallets.byId
+      userId: state.core.account.id,
+      wallets: state.ui.wallets.byId,
+      disklet: state.core.disklet
```

Comment:
Can you confirm with @swansontec that this Disklet is only local to the device and account agnostic.

Reply:
Yes it is